cvat-core/src/lambda-manager.ts

```diff
@@ -18,18 +24,26 @@ class LambdaManager {
     }
 
     async list(): Promise<MLModel[]> {
-        if (Array.isArray(this.cachedList)) {
-            return [...this.cachedList];
-        }
-
-        const result = await serverProxy.lambda.list();
+        let lambdaFunctions = [];
+        let functions = [];
+        try {
+            // lambda.list returns error if list is empty, but we should work with models anyway
+            lambdaFunctions = await serverProxy.lambda.list();
```

Comment:
1. why we have error in case with empty list? Do you mean 404?
2. it ignores all the errors, not sure it is correct
3. better to handle on server-proxy because it looks like server-specific thing